website/frontend/config/urls.js

```diff
@@ -33,3 +33,5 @@ export const CAREERS_URL = `${WEBSITE_BASE_URL}/career/`;
 export const DEPARTMENTS_URL = `${WEBSITE_BASE_URL}/departments/`;
 
 export const TEAMS_URL = `${WEBSITE_BASE_URL}/team/`;
+
+export const PROD_NETMANAGER_URL = stripTrailingSlash(process.env.REACT_NETMANAGER_BASE_URL.replace("/api/v1/", ""));
```

Comment:
Could we give this value a more appropriate name such as `NETMANAGER_URL` instead of `PROD_NETMANAGER_URL` cause this can be misleading especially since we also have a different value for it in staging.
or we can also have both `PROD_NETMANAGER_URL` and `STAGE_NETMANAGER_URL` if it is easier but that is not preferred.

Reply:
I have used NETMANAGER_URL @123MwanjeMike